python/lsst/afw/cameraGeom/testUtils.py

```diff
@@ -104,6 +106,9 @@ def __init__(self,
             CameraSys(TAN_PIXELS, self.name): pixelToTanPixel,
             CameraSys(ACTUAL_PIXELS, self.name): afwGeom.RadialXYTransform([0, 0.95, 0.01]),
         }
+        if crosstalk is None:
+            crosstalk = [[0.0 for _ in range(numAmps)] for _ in range(numAmps)]
```

Comment:
This is a crosstalk matrix that is all zero. I could put other dummy values instead of zeros, but it wouldn't add anything to the test.